src/AppInstallerCLICore/Workflows/InstallFlow.cpp

```diff
@@ -443,6 +458,7 @@ namespace AppInstaller::CLI::Workflow
     void InstallSinglePackage(Execution::Context& context)
     {
         context <<
+            Workflow::DisplayInstallWarnings <<

```

Comment:
Need to update InstallMultiplePackages as well for import, dependencies, upgrade all etc

Reply:
Added to `InstallMultiplePackages`